src/helpers/test-upgrades/SynthPermissioner.sol

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: MIT
```

Comment:
This is necessary to upgrade the old Synthesizer for which we cannot change the storage layout, correct?
Can we add this as a comment here?